### PR TITLE
Microsoft.dotnetUninstallTool version 4.80.595

### PR DESCRIPTION
**Pull request by dgellow**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36291)

### CODE REVIEW

**Comment by dgellow** [comment]
Manifest for the .Net SDK uninstall tool from https://github.com/dotnet/cli-lab.

It's the first time I ever create a winget manifest, it's very likely I missed a few things. I used the wingetcreate tool, and tried the manifest locally to install the msi.

File: manifests/m/Microsoft/dotnetUninstallTool/1.5.255402/Microsoft.dotnetUninstallTool.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnetUninstallTool
+PackageVersion: 1.5.255402
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Microsoft .Net SDK Uninstall Tool
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetUninstallTool/1.5.255402/Microsoft.dotnetUninstallTool.locale.en-US.yaml:8]:
```suggestion
PackageName: Microsoft .NET Core SDK Uninstall Tool
```

File: manifests/m/Microsoft/dotnetUninstallTool/1.5.255402/Microsoft.dotnetUninstallTool.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnetUninstallTool
+PackageVersion: 1.5.255402
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetUninstallTool/1.5.255402/Microsoft.dotnetUninstallTool.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.80.595
```

File: manifests/m/Microsoft/dotnetUninstallTool/1.5.255402/Microsoft.dotnetUninstallTool.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnetUninstallTool
+PackageVersion: 1.5.255402
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetUninstallTool/1.5.255402/Microsoft.dotnetUninstallTool.yaml:5]:
```suggestion
PackageVersion: 4.80.595
```

File: manifests/m/Microsoft/dotnetUninstallTool/1.5.255402/Microsoft.dotnetUninstallTool.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnetUninstallTool
+PackageVersion: 1.5.255402
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetUninstallTool/1.5.255402/Microsoft.dotnetUninstallTool.installer.yaml:5]:
```suggestion
PackageVersion: 4.80.595
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by dgellow** [comment]
@ItzLevvie Where does the version number 4.80.595 come from? I used 1.5.255402 because that's the version mentioned in the release page: https://github.com/dotnet/cli-lab/releases.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> https://github.com/microsoft/winget-pkgs/pull/36291#issuecomment-981122490

`PackageVersion` must match Control Panel.

**Comment by dgellow** [comment]
Should I also change the directory name then? It is currently `manifests/m/Microsoft/dotnetUninstallTool/1.5.255402`. Should I rename it to `m/Microsoft/dotnetUninstallTool/4.80.595`? I have to say, I do not understand why there are two different version numbers, that should maybe be clarified with `dotnet-core-uninstall` authors.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Should I also change the directory name then? It is currently manifests/m/Microsoft/dotnetUninstallTool/1.5.255402. Should I rename it to m/Microsoft/dotnetUninstallTool/4.80.595?

Yes

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**